src/vs/workbench/contrib/preferences/browser/settingsTree.ts

```diff
@@ -1209,6 +1220,10 @@ abstract class AbstractSettingObjectRenderer extends AbstractSettingRenderer imp
 			} else {
 				template.objectDropdownWidget!.setValue(newItems);
 			}
+
+			if (template.renderValidations) {
```

Comment:
If we are going to add a separate function for it, seems like it would make sense to use onChange on the template to simplify? I don't know if there is a good reason we didn't do that originally?